src/pages/how-to/configure-ipv6-for-peers.mdx

```diff
@@ -0,0 +1,63 @@
+
+# Configure IPv6 support for your peers
+
+In addition to IPv4, Netbird also supports both establishment of connections over IPv6 as well as assignment of IPv6 addresses 
+to peers within the network.
+
+<Note>
+    Assignment of IPv6 addresses is available for NetBird [TODO] or later.
```

Comment:
Before merging,  this [TODO] must be replaced with the version of netbird that introduces the changes of netbirdio/netbird#1459.